pkg/plugins/manager/signature/manifest.go

```diff
@@ -177,6 +180,9 @@ func Calculate(mlog log.Logger, plugin *plugins.Plugin) (plugins.Signature, erro
 	var unsignedFiles []string
 	for _, f := range pluginFiles {
 		if _, exists := manifestFiles[f]; !exists {
+			if runningWindows && plugin.IsRenderer() && f == "chrome-win/debug.log" {
```

Comment:
Even though the tests serve as an explanation, would it make sense to add a comment above this line as well explaining why it is needed? (Asking as I have a 6th sense that new devs might don't understand it immediately). E.g.: `"Ignoring unsigned Chromium debug.log so it doesn't invalidate the signature for Renderer plugin running on Windows"`

Reply:
Could this file be made executable? If that's the case we should probably fail to load the plugin

Reply:
My Windows knowledge is lacking here, but maybe that's possible? We could fail this scenario if we have a valid check for Windows, but I'm not sure Go provides this out of the box. Any ideas?

Reply:
@jmatosgrafana wanted to loop you in here to weigh in if you don't mind!

Reply:
@wbrowne LGTM from a security point of view as long as the plugin never tries to execute `debug.log` (a bit like `MANIFEST.txt`)